### PR TITLE
Update UG for Email and Cap field

### LINKED ISSUE #195: [PE-D][Tester C] Fields Description - LOCAL_PART and DOMAIN_PART a little confusing
you could add a hyperlink to the glossary or explain beforehand what these mean. If i read from top down, i would be lost as these terms are completely new

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: ReubenChay/ped#2

### PR DESCRIPTION
**Pull request by maryjess**
Changes made:
- Clarify `LOCAL_PART` and `DOMAIN` explanation for `Email` field -- Resolves #195 
- Clarify explanation for `Cap` field that only accepts positive values and rounds them -- Related to #253

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -86,25 +86,30 @@ InternConnect is a **desktop app for managing internship applicants, optimized f
 | **Job ID**  | `ji`      | 10           | Can only contain upper case alphanumeric characters and spaces |
 
 * `LOCAL_PART` can only contain alphanumeric characters and any of `+` `_` `.` `-`, but must not start or end with these special characters
-* `DOMAIN` can be made up of numerous `DOMAIN_LABEL` separated by `.`, with the last `DOMAIN_LABEL` being at least 2 characters long
-* `DOMAIN_LABEL` can only contain alphanumeric characters separated by `-`, but must start and end with alphanumeric characters
+* `DOMAIN` refers to the email domain that can be made of numerous `DOMAIN_LABEL`
+    * `DOMAIN_LABEL` can be separated by `.`
+    * `DOMAIN_LABEL` can only contain alphanumeric characters separated by `-`
```
Comment (cliftonfelix) [inline docs/UserGuide.md:91]: May I know whats the difference between line 90 and 91? It sounds contradicting

**Review by cliftonfelix — CHANGES_REQUESTED** [review]
I also wanna check have you figured out why the previous message is wrong?

File: docs/UserGuide.md (hunk 1 above)
Comment (maryjess) [reply docs/UserGuide.md:91]: `DOMAIN_LABEL` refers to one part of the `DOMAIN`, and each `DOMAIN_LABEL` is separated by periods. But for one `DOMAIN_LABEL` itself, should only contain alphanumeric characters and hyphens, and not started or ended by a hyphen. It is perhaps better to clarify by adding "Each" here for line 91, thanks for pointing out!

**Comment by maryjess** [comment]
> I also wanna check have you figured out why the previous message is wrong?

There is nothing wrong with the previous message for email, I just clarified it so that the user can understand better.

In #197, I think there was a misunderstanding as to why the email failed to be processed -- it is due to the local part (the mistake is located at illegal symbol `@` and `!` in local part), and nothing wrong with the domain

**Review by lulucopter — APPROVED** [review]
LGTM

**Review by cliftonfelix — APPROVED** [review]
LGTM

**Pull request merged**